### PR TITLE
Adding checks for starting mux for 2.2.0

### PR DESCRIPTION
**Pull request by ewolinetz**
cc @jupierce 

This is to address #4007 
I was unable to recreate the original issue locally, however this should prevent it

### CODE REVIEW

File: roles/openshift_logging/tasks/start_cluster.yaml [hunk 1]
```diff
@@ -39,7 +39,10 @@
   with_items: "{{ mux_dc.results.results[0]['items'] | map(attribute='metadata.name') | list }}"
   loop_control:
     loop_var: object
-  when: openshift_logging_use_mux
+  when:
```
Comment (richm) [inline roles/openshift_logging/tasks/start_cluster.yaml]: - mux_dc is defined

Comment (ewolinetz) [inline roles/openshift_logging/tasks/start_cluster.yaml]: it should be defined, even if the task skips it will register the output

Comment (richm) [inline roles/openshift_logging/tasks/start_cluster.yaml]: ok - ansible conditionals are mysterious . . .

**Comment by ewolinetz** [comment]
aos-ci-test

**Comment by sdodson** [comment]
[merge]

**Comment by ewolinetz** [comment]
Created issue for error seen on merge job -- https://github.com/openshift/origin/issues/13931

**Comment by ewolinetz** [comment]
flaked on https://github.com/openshift/origin/issues/13931
[merge]

**Pull request merged**